internal/server/v1/firehose/actions.go

```diff
@@ -64,14 +79,28 @@ func (api *firehoseAPI) handleScale(w http.ResponseWriter, r *http.Request) {
 }
 
 func (api *firehoseAPI) handleStart(w http.ResponseWriter, r *http.Request) {
-	var reqBody struct{}
+	var reqBody struct {
+		StopTime *time.Time `json:"stop_time,omitempty"`
+	}
 	if err := utils.ReadJSON(r, &reqBody); err != nil {
 		utils.WriteErr(w, err)
 		return
 	}
 
+	// Ensure that the URN refers to a valid firehose resource.
 	urn := chi.URLParam(r, pathParamURN)
-	updatedFirehose, err := api.executeAction(r.Context(), urn, actionStart, reqBody)
+	existingFirehose, err := api.getFirehose(r.Context(), urn)
+	if err != nil {
+		utils.WriteErr(w, err)
+		return
+	}
+	// for LOG sinkType, updating stop_time
+	if existingFirehose.Configs.EnvVars[confSinkType] == logSinkType {
+		t := time.Now().Add(logSinkTTL)
+		reqBody.StopTime = &t
```

Comment:
instead of using request body, we can use a new variable to store it.

also, have you tested if we can change stop date on `ApplyActionRequest` API in entropy?